platform/linux-generic/include/odp_chksum_internal.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP checksum - implementation internal
+ */
```

Comment:
Internal files are not included into doxygen documentation. So, these 5 lines of doxygen documentation are unnecessary and can be removed.

Reply:
ack